docs/UsersGuide.asciidoc

```diff
@@ -1010,11 +1010,15 @@ Example for `_DEPRIORITIZEBUILD` and `_DEPRIORITIZE_LIMIT`.
 
 [source,sh]
 --------------------------------------------------------------------------------
-openqa-cli api -X POST isos ISO=my_iso.iso DISTRI=my_distri FLAVOR=sweet \
-         ARCH=my_arch VERSION=42 BUILD=1234 \
+openqa-cli api -X POST isos async=0 ISO=my_iso.iso DISTRI=my_distri \
+         FLAVOR=sweet ARCH=my_arch VERSION=42 BUILD=1234 \
          _DEPRIORITIZEBUILD=1 _DEPRIORITIZE_LIMIT=120 \
```

Comment:
Given that we show the most simple examples already in before with - if you follow my suggestion - the default then I would be ok with explicitly repeating the default setting `async=0` as you do here

Reply:
Removed it in the simpler examples.